inference-engine/thirdparty/clDNN/src/include/pass_manager.h

```diff
@@ -102,9 +102,9 @@ class graph_initializations : public base_pass {
 
 private:
     void run(program_impl& p) override;
-    void replace_nodes(program_impl& p);
-    void handle_lstm(program_impl& p);
-    void handle_dynamic_lstm(program_impl& p);
+    void handle_split_node(program_impl& p, std::shared_ptr<program_node> node);
```

Comment:
I think the parameter type should be changed to `split_node& node`, lstm_node, etc

Reply:
done